docs/third-party.md

```diff
@@ -0,0 +1,47 @@
+# Third-party Licenses
+
+Several free and open-source pieces of software have been used in the
+construction of Antrea. A few require that we include their license agreeements
+within our product. We have included all third-party software and their their
+accompanying licenses in the list below. We are grateful to all the wonderful
+authors of the software that has helped make Antrea possible.
+
+Third party software that is part of Antrea:
+
+- [Third-party Licenses](#third-party-licenses)
+  - [Golang](#golang)
+
+## Golang
```

Comment:
I don't mind adding this, but we do not distribute any part of the Golang project in source form or binary form (the Go compiler for example is not included in any of our Docker images).

Reply:
Also added by mistake. I will revert this.